### ISSUE TITLE
[BUG] Bloxstrap dont open

### ISSUE THREAD

**Issue by fafabryt** [bloxstraplabs/bloxstrap#3503]
### Acknowledgement of preliminary instructions

- [X] I have read the preliminary instructions, and I am certain that my problem has not already been addressed.
- [X] I have thoroughly looked through the available Wiki articles and could not find a solution to my problem.
- [X] I am using the latest version of Bloxstrap.
- [ ] I did not answer truthfully to all the above checkboxes.

### Bloxstrap Version

v2.8.0

### What problem did you encounter?

It's been a few weeks since my bloxstrap doesn't work, I go into normal roblox and it open normally, when I open bloxtrap an error appears and sends me to normal roblox for no reason

### Bloxstrap Log

```text
2024-10-27T17:40:44Z [App::OnStartup] Starting Bloxstrap v2.8.0
2024-10-27T17:40:44Z [App::OnStartup] Compiled Wednesday, 16 October 2024 at 6:44:37 AM from commit 42ffec26cf2e080fe22210305a08fcb286aeb577 (tag/v2.8.0)
2024-10-27T17:40:44Z [App::OnStartup] Loaded from %UserProfile%\AppData\Local\Bloxstrap\Bloxstrap.exe
2024-10-27T17:40:44Z [App::OnStartup] Temp path is %UserProfile%\AppData\Local\Temp\Bloxstrap
2024-10-27T17:40:44Z [App::OnStartup] WindowsStartMenu path is %UserProfile%\AppData\Roaming\Microsoft\Windows\Start Menu\Programs
2024-10-27T17:40:45Z [Logger::Initialize] Initializing at %UserProfile%\AppData\Local\Bloxstrap\Logs\Bloxstrap_20241027T174045Z.log
2024-10-27T17:40:45Z [Logger::Initialize] Finished initializing!
2024-10-27T17:40:45Z [JsonManager<Settings>::Load] Loading from %UserProfile%\AppData\Local\Bloxstrap\Settings.json...
2024-10-27T17:40:46Z [JsonManager<Settings>::Load] Loaded successfully!
2024-10-27T17:40:46Z [JsonManager<State>::Load] Loading from %UserProfile%\AppData\Local\Bloxstrap\State.json...
2024-10-27T17:40:46Z [JsonManager<State>::Load] Loaded successfully!
2024-10-27T17:40:46Z [FastFlagManager::Load] Loading from %UserProfile%\AppData\Local\Bloxstrap\Modifications\ClientSettings\ClientAppSettings.json...
2024-10-27T17:40:46Z [FastFlagManager::Load] Loaded successfully!
2024-10-27T17:40:46Z [FastFlagManager::SetValue] Setting of 'FLogNetwork' to '7' is pending
2024-10-27T17:40:55Z [LaunchHandler::LaunchRoblox] Initializing bootstrapper
2024-10-27T17:40:55Z [LaunchHandler::LaunchRoblox] Initializing bootstrapper dialog
2024-10-27T17:40:55Z [Bootstrapper::Run] Running bootstrapper
2024-10-27T17:40:55Z [Bootstrapper::SetStatus] Conectando-se ao Roblox...
2024-10-27T17:40:55Z [Deployment::TestConnection<https://setup.rbxcdn.com>] Connecting...
2024-10-27T17:40:55Z [HttpClientLoggingHandler::ProcessRequest] GET https://setup.rbxcdn.com/versionStudio
2024-10-27T17:40:56Z [Deployment::InitializeConnectivity] Testing connectivity...
2024-10-27T17:40:56Z [HttpClientLoggingHandler::ProcessResponse] 200 OK https://setup.rbxcdn.com/versionStudio
2024-10-27T17:40:56Z [Deployment::InitializeConnectivity] Got https://setup.rbxcdn.com as the optimal base URL
2024-10-27T17:40:56Z [Bootstrapper::Run] Connectivity check finished
2024-10-27T17:40:56Z [Bootstrapper::CheckForUpdates] Checking for updates...
2024-10-27T17:40:56Z [HttpClientLoggingHandler::ProcessRequest] GET https://api.github.com/repos/bloxstraplabs/bloxstrap/releases/latest
2024-10-27T17:40:57Z [HttpClientLoggingHandler::ProcessResponse] 200 OK https://api.github.com/repos/bloxstraplabs/bloxstrap/releases/latest
2024-10-27T17:40:57Z [Bootstrapper::CheckForUpdates] No updates found
2024-10-27T17:40:57Z [Bootstrapper::GetLatestVersionInfo] Got channel as production
2024-10-27T17:40:57Z [Deployment::GetInfo] Getting deploy info for channel production
2024-10-27T17:40:57Z [HttpClientLoggingHandler::ProcessRequest] GET https://clientsettingscdn.roblox.com/v2/client-version/WindowsPlayer
2024-10-27T17:40:57Z [HttpClientLoggingHandler::ProcessResponse] 200 OK https://clientsettingscdn.roblox.com/v2/client-version/WindowsPlayer
2024-10-27T17:40:57Z [RegistryKeyEx::SetValueSafe] Writing '' to HKEY_CURRENT_USER\SOFTWARE\ROBLOX Corporation\Environments\RobloxPlayer\Channel\www.roblox.com
2024-10-27T17:40:57Z [HttpClientLoggingHandler::ProcessRequest] GET https://setup.rbxcdn.com/version-4ffdeb3e393e469e-rbxPkgManifest.txt
2024-10-27T17:40:57Z [HttpClientLoggingHandler::ProcessResponse] 200 OK https://setup.rbxcdn.com/version-4ffdeb3e393e469e-rbxPkgManifest.txt
2024-10-27T17:40:57Z [Bootstrapper::SetStatus] Aplicando modificações ao Roblox...
2024-10-27T17:40:57Z [Bootstrapper::ApplyModifications] Checking file mods...
2024-10-27T17:40:57Z [Bootstrapper::ApplyModifications] ClientSettings\ClientAppSettings.json already exists in the version folder, and is a match
2024-10-27T17:40:57Z [JsonManager<State>::Save] Saving to %UserProfile%\AppData\Local\Bloxstrap\State.json...
2024-10-27T17:40:57Z [JsonManager<State>::Save] Save complete!
2024-10-27T17:40:57Z [Bootstrapper::ApplyModifications] Finished checking file mods
2024-10-27T17:40:57Z [Bootstrapper::SetStatus] Inicializando {product}...
2024-10-27T17:40:57Z [Bootstrapper::StartRoblox] Started Roblox (PID 4648), waiting for start event
2024-10-27T17:41:02Z [Bootstrapper::StartRoblox] Start event not signalled, implying successful launch
2024-10-27T17:41:07Z [Bootstrapper::StartRoblox] Unable to identify log file
2024-10-27T17:41:07Z [Frontend::ShowMessageBox] Roblox falhou ao iniciar.

Leia as seguintes informações de ajuda, que serão abertas no seu navegador quando você fechar esta caixa de diálogo.
2024-10-27T17:41:20Z [LaunchHandler::LaunchRoblox] Bootstrapper task has finished
2024-10-27T17:41:20Z [App::Terminate] Terminating with exit code 0 (ERROR_SUCCESS)
```

**Comment by fafabryt** [comment]
> What do you mean by "sends me to normal roblox"?

Open normal Roblox, without any bloxstrap modifications and not optimized 
While I get a message from bloxstrap saying that it was not possible to open bloxstrap and sends me to the Wiki talking about possible solutions

**Comment by Wanwood1** [comment]
> > What do you mean by "sends me to normal roblox"?
> 
> Open normal Roblox, without any bloxstrap modifications and not optimized While I get a message from bloxstrap saying that it was not possible to open bloxstrap and sends me to the Wiki talking about possible solutions

Maybe try switching to bloxstrap?
https://github.com/bloxstraplabs/bloxstrap/wiki/Switching-between-Roblox-and-Bloxstrap

**Comment by pizzaboxer** [comment]
Assuming issue is resolved with v2.8.1

**Comment by fafabryt** [comment]
> Assuming issue is resolved with v2.8.1

when will you update?

**Comment by fafabryt** [comment]
> Assuming issue is resolved with v2.8.1

when will you update?
